proposals/function-pointers.md

```diff
@@ -11,122 +11,157 @@ potential implementation of the feature):
 
 https://github.com/dotnet/csharplang/issues/191
 
-This is an alternate design propsoal to [compiler intrinsics](https://github.com/dotnet/csharplang/blob/master/proposals/intrinsics.md)
+This is an alternate design propsoal to [compiler intrinsics]
+(https://github.com/dotnet/csharplang/blob/master/proposals/intrinsics.md)
 
 ## Detailed Design 
 
-### funcptr
-The language will allow for the declaration of function pointers using the `funcptr` contextual keyword. The 
-declaration and usage of function pointers closely resemble that of `delegate`:
+### Function pointers
+The language will allow for the declaration of function pointers using the `func*` syntax. The full syntax is described
+in detail in the next section but it is meant to resemble the syntax used by `delegate` declarations.
 
 ``` csharp
 unsafe class Example {
     delegate void DAction(int a);
-    funcptr void FAction(int a);
 
-    void Example(DAction d, FAction f) {
+    void Example(DAction d, func* void(int) f) {
         d(42);
         f(42);
     }
 }
 ```
 
 These types are represented using the function pointer type as outlined in ECMA-335. This means invocation
-of a `funcptr` will use `calli` where invocation of a `delegate` will use `callvirt` on the `Invoke` method.
+of a `func*` will use `calli` where invocation of a `delegate` will use `callvirt` on the `Invoke` method.
 Syntactically though invocation is identical for both constructs.
 
-The `calli` instruction requires the calling convention be specified as a part of the invocation. The default 
-for `funcptr` will be managed. Alternate forms can be specified by adding the appropriate modifier after the 
-`funcptr` keyword: `cdecl`, `fastcall`, `stdcall`, `thiscall` or `winapi`. Example:
+The ECMA-335 definition of method pointers includes the calling convention as part of the type signature (section 7.1).
+The default calling convention will be `managed `. Alternate forms can be specified by adding the appropriate modifier 
+after the `func*` syntax: `cdecl`, `fastcall`, `stdcall`, `thiscall` or `winapi`. Example:
```

Comment:
Will explicitly specifying `managed` as the calling convention be supported?

Reply:
Please read the section on calling conventions where this is called out.

Reply:
I saw that as I got further down. I was just expecting it to be listed here as well, since it is relevant 😄